README.rst

```diff
@@ -2,8 +2,17 @@
                             Tarantool C client libraries
 -------------------------------------------------------------------------------
 
-.. image:: https://travis-ci.org/tarantool/tarantool-c.svg?branch=master
-    :target: https://travis-ci.org/tarantool/tarantool-c
+.. image:: https://img.shields.io/github/workflow/status/tarantool/tarantool-c/fast_testing/master?label=Linux%20test
+    :target: https://github.com/tarantool/tarantool-c/actions/workflows/fast_testing.yml?query=branch%3Amaster
+    :alt: GitHub Workflow Status (branch)
```

Comment:
The alt text does not look useful: it is the same for all badges. I would make it the same as label in the URL.

Reply:
Good. I've returned `:alt:` with text similar to label.